### ISSUE TITLE
iPattern error - please advise

### ISSUE THREAD

**Issue by sprakashUTH** [HaoKeLab/ensembleCNV#10]
I'm running iPattern using the ilmn.sh wrapper. I processed the Final Report file using the ensembleCNV script. The analysis completed all steps up until aggregating sample level calls before terminating with this error:

$ more chr21.p.EBAV10.int.27934.Rerr
Loading required package: ppc
Loading required package: cluster
Read 4 records
Error: Window size must be no larger than the total number of probes!
Execution halted
 
This is the only one of my .Rerr files that shows a fatal error. All other output files showed no errors or (5 others) contained warnings. Because of this error, execution halted before the final output call files were generated.
How can I deal with this error?

**Comment by diyanathrs** [comment]
Hi, 
Did you figure out what went wrong here? 

Thanks in advance

**Comment by sprakashUTH** [comment]
Yes. Ultimately we decided not to use iPattern because it performed far worse than other algorithms on our datasets.

From: Diyanath Ranasinghe ***@***.***>
Date: Tuesday, August 8, 2023 at 4:41 AM
To: HaoKeLab/ensembleCNV ***@***.***>
Cc: Prakash, Siddharth K ***@***.***>, Author ***@***.***>
Subject: Re: [HaoKeLab/ensembleCNV] iPattern error - please advise (#10)
External: Increase caution when handling links and attachments.


Hi,
Did you figure out what went wrong here?

Thanks in advance

**Comment by diyanathrs** [comment]
Thank you for your reply. 
Was it due to incorrect window size or loss of probe information in the signal intensity file? 

Yes, I too think iPattern perform poorly compared to other algorithms we use.

Thank you,
Dean
